templates/cinder/bin/init.sh

```diff
@@ -73,8 +71,7 @@ if [ -n "$TRANSPORTURL" ]; then
   crudini --set ${SVC_CFG_MERGED} DEFAULT transport_url $TRANSPORTURL
 fi
 crudini --set ${SVC_CFG_MERGED} database connection mysql+pymysql://${DBUSER}:${DBPASSWORD}@${DBHOST}/${DB}
-crudini --set ${SVC_CFG_MERGED} keystone_authtoken password $CINDERPASSWORD
-# TODO: nova password
-#crudini --set ${SVC_CFG_MERGED} nova password $NOVAPASSWORD
+crudini --set ${SVC_CFG_MERGED} keystone_authtoken password $PASSWORD
+crudini --set ${SVC_CFG_MERGED} nova password $PASSWORD
```

Comment:
it is assumed to be a CinderPassword, but used for nova password as well. Would be nice to clarify the password equality requirements, or the like

Reply:
Do you mind clarifying what you expect ? Do you want me to add a note (though the node has not yet been added to any other operators so I tend to avoid putting it only in cinder operator)

Reply:
having something noted about different passwords used would be nice, thanks

Reply:
Just to avoid my misunderstanding. This implementation does not mean that we should use the same password for different service users.

As we are using the same cinder user, it's quite reasonable to use the same password. You can check the config file template to find out which user is used. I can add a note but if we add one then we have to add the same to every operator doing this. Do you want me to update all operators ? I see even nova operator does not contain such a note.

Reply:
I think there's an effort to remove crudini from the init script (see #108): it's still in progress and it can be rebased on this change when it's ready.
Adding @ASBishop for awareness, but I don't see big problems rebasing his change on top of this.

Reply:
We should probably document somewhere the expectations on the cinder user.

Reply:
We probably can discuss this in the next sync but I think we should document the general design (use own service user to interact with the other services) somewhere more common instead of adding comments in individual files.

Reply:
I didn't mean as a comment here, I was thinking more in terms of documentation.